lib/errors.js

```diff
@@ -9,6 +9,7 @@ function ValidationError(rule, params, info) {
     const valuePath = ['value'].concat(opts.path);
     const key = valuePath[valuePath.length - 1];
 
+    this.stack = (new Error()).stack;
```

Comment:
This slows things down a _lot_. It might make sense to have stack-attachment as a validation option that's enabled for things like assert, and disabled otherwise.